attendance-backend/src/controllers/StudentController.js

```diff
@@ -0,0 +1,22 @@
+const {
+  successResponseWithData,
+} = require('../helpers/response');
+
+module.exports.getStudentDetailsController = [async (req, res) => {
+  delete req.user.password;
+  successResponseWithData(res, res);
```

Comment:
Since we are getting the details of the current user the preferred path would be ` /students/me` instead of `/students/details`.